components/filter/filter.js

```diff
@@ -52,6 +52,11 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 			 * @type {boolean}
 			 */
 			opened: { type: Boolean, reflect: true },
+			/**
+			 * Optional override for the button text used for a multi-dimensional filter
+			 * @type {string}
+			 */
+			text: { type: String },
```

Comment:
I wish we'd consider removing these in favour of the Daylight Site, but until we make that call this needs to be duplicated here:
https://github.com/BrightspaceUI/core/tree/main/components/filter#properties

Reply:
Ah right. If anyone wants to take this to a vote, my vote would be "do not maintain this in the readme as well".  I don't see a public Daylight site as a blocker for that... the way we make breaking changes constantly already means we aren't ready to actually support this externally

Reply:
I agree -- I'm in that camp as well.